eogrow/pipelines/batch_to_eopatch.py

```diff
@@ -61,19 +57,24 @@ class Schema(Pipeline.Schema):
             ),
             example="\"[info['date'] for info in json.loads(userdata['metadata'])]\"",
         )
+
+        mapping: List[FeatureMappingSchema] = Field(
+            description="A list of mapping from batch files into EOPatch features."
+        )
+
+        @validator("mapping")
+        def check_something_is_converted(cls, value: list, values: RawSchemaDict) -> list:
```

Comment:
I find this name confusing and not really sure what the purpose of the validator is, could you suggest a better name?

Reply:
I am almost feeling like this should just be removed. Perhaps `check_nonempty_input`?

Reply:
it definitely sounds better, yes

Reply:
done